docs/DeveloperGuide.md

```diff
@@ -257,37 +257,36 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* have large number of students to manage
+* accept fixed rate hour pay as a tutor, no matter which subject
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
```

Comment:
Probably should follow the target user profile that we submitted

![image](https://user-images.githubusercontent.com/35413456/223060988-7f9451d3-4a5d-43e1-a82f-49e8d8f4e5cf.png)